code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -335,6 +338,7 @@ type Info struct {
 type StorageNode struct {
 	ID                      string                 `json:"id"`
 	BaseURL                 string                 `json:"url"`
+	ProviderType            spenum.Provider        `json:"provider_type"`
```

Comment:
storage node is always a blobber, why to add this field?

Reply:
As above